application/application.go

```diff
@@ -25,3 +27,21 @@ func checkAllBundleContentsAreApproved(contents []models.BundleContent) bool {
 	}
 	return true
 }
+
+type Bundlestore interface {
```

Comment:
This is not required as the datastore which already exists in the StateMachineBundleAPI can be used to retrieve the data